popup/popup.js

```diff
@@ -397,11 +397,11 @@ Object.assign(handleEvent, {
   },
 
   toggleMenu(event) {
+    document.documentElement.classList.toggle('menu-shown')
```

Comment:
What does it do? I think this won't work as expected.

Reply:
Won't work at all, because it doesn't do anything. It's a leftover from different methods I was testing to fix the Firefox issue mentioned above.